src/gc.rs

```diff
@@ -0,0 +1,76 @@
+use crate::Sodg;
+use anyhow::{Context, Result};
+use std::collections::VecDeque;
+
+impl Sodg {
+    pub fn collect(&mut self, v: u32) -> Result<()> {
```

Comment:
@UARTman we should not make it part of the public API. The GC mechanism should be invisible to the end-user.

Reply:
@yegor256 the easiest solution would be to make it `pub(crate)` so that it can be used from `ops.rs` and elsewhere, but not by the users.

Reply:
@UARTman this would work